pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/JdbiClient.java

```diff
@@ -132,7 +132,7 @@ int deleteByAuth0ClientIdAndAuth0TenantId(
             @Bind("auth0TenantId") long auth0TenantId
     );
 
-    @SqlQuery("SELECT c.*, (select auth0_domain from auth0_tenant as t where t.auth0_tenant_id = c.auth0_tenant_id) as auth0_domain"
+    @SqlQuery("SELECT c.*, (select t.auth0_domain from auth0_tenant as t where t.auth0_tenant_id = c.auth0_tenant_id) as auth0_domain"
```

Comment:
since column: autho_domain doesn't exist in both the tables, alias is not a must. 
Its little cleaner to have it though